src/peft/peft_model.py

```diff
@@ -215,12 +215,12 @@ def save_pretrained(
             is_main_process (`bool`, *optional*):
                 Whether the process calling this is the main process or not. Will default to `True`. Will not save the
                 checkpoint if not on the main process, which is important for multi device setups (e.g. DDP).
-            convert_pissa_to_lora (`str`):
-                The path to the initialized PiSSA adapter, which is obtained after initializing the model with PiSSA
-                and before performing any training. When `convert_pissa_to_lora` is not None, the difference in PISSA
+            convert_mutated_to_lora (`Literal["olora", "pissa"]`):
```

Comment:
Regarding the name, I think we have to find something better. Of course, we cannot keep the old name but at the same time, I'm afraid that the new name could confuse users. I don't have really good ideas, but something like `path_initial_model_for_weight_conversion` could be more fitting, WDYT?

Moreover, the type annotation is wrong, isn't it? The argument is the path to the base model, not the initialization scheme, or `None`, so `Optional[str]` would be correct.

Reply:
I also had really hard time coming up with a name. Just don't know what to call it. And yes type should be fixed.